mustache_test.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+    . "github.com/onsi/ginkgo"
+    . "github.com/onsi/gomega"
+    "github.com/onsi/gomega/gexec"
+    "github.com/onsi/gomega/gbytes"
+    "os/exec"
+)
+
+var _ = Describe("Mustache", func() {
+    successDecodeOutput := map[string]interface{}{"key":"val"}
+    expectedTemplate := "my name is Montoooorb69 but you can call me Monica Rachel Pheobe.\n" +
+        "I come from Cloooooooooooooooo in the Florbatorb galaxy.\n" +
+        "I would like to Drink earth liquids, Kill the producers of 3rd Rock From The Sun, Go to Disney World.\n\n" +
+        "Thank you,\nMontoooorb69"
+
+    assertExampleSuccess := func(path string) {
+        command := exec.Command(path)
+        session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+        Expect(err).ShouldNot(HaveOccurred())
+        Eventually(session.Out).Should(gbytes.Say(expectedTemplate))
+        Eventually(session).Should(gexec.Exit(0))
+    }
+
+    Describe("Cli", func() {
```

Comment:
I think this Describe should be `Describe("When CLI is executed"` so that the `It`s will read more like a sentence.